scripts/scr_weapon/scr_weapon.gml

```diff
@@ -1308,98 +1372,98 @@ global.weapons = {
 	},
 	"Flamestorm Cannon Sponsons": {
 		"attack": {
-			"standard": 600,
-			"master_crafted": 660,
-			"artifact": 720
+			"standard": 750,
+			"master_crafted": 850,
+			"artifact": 900
 		},
 		"description": "A huge vehicle-mounted flamethrower cannon, the heat produced by this terrifying weapon can melt armoured ceramite.",
 		"abbreviation": "FlmstCnns",
 		"melee_hands": 0,
 		"ranged_hands": 0,
 		"ammo": 6,
-		"range": 2,
-		"spli": 3,
+		"range": 4.1,
+		"spli": 30,
 		"arp": 1,
 		"tags": ["flame","heavy_ranged", "vehicle", "sponson"]
```

Comment:
As you've changed the Flamestorm turret to have negative ap, I assume this one should have it as well.
```suggestion
		"spli": 30,
		"arp": -1,
		"tags": ["flame","heavy_ranged", "vehicle", "sponson"]
```